meta-refkit/conf/bblayers.conf.sample

```diff
@@ -15,6 +15,7 @@ REFKIT_LAYERS = " \
   ##OEROOT##/../meta-iotqa \
   ##OEROOT##/../meta-security-isafw \
   ##OEROOT##/../meta-intel-realsense \
+  ##OEROOT##/../meta-clang \
```

Comment:
Yes and also ```LAYER_CONF_VERSION``` in layer.conf. This ensures that users update their build/conf/bblayers.conf before bitbake parses the recipes with these changes included. The recipe parsing fails with an error because this PR brings a bbappend but the corresponding bb is in meta-clang (which is unlikely enabled unless we force it with the conf version change).